virtual_rainforest/models/abiotic/abiotic_model.py

```diff
@@ -113,25 +113,14 @@ def from_config(cls, data: Data, config: dict[str, Any]) -> AbioticModel:
         # Assume input is valid until we learn otherwise
         valid_input = True
         try:
-            raw_interval = pint.Quantity(config["abiotic"]["model_time_step"]).to(
-                "minutes"
+            start_time, update_interval = extract_model_time_details(
+                config, cls.model_name
             )
-            # Round raw time interval to nearest minute
-            update_interval = timedelta64(int(round(raw_interval.magnitude)), "m")
-            start_time = datetime64(config["core"]["timing"]["start_time"])
-            soil_layers = config["abiotic"]["soil_layers"]
-            canopy_layers = config["abiotic"]["canopy_layers"]
-        except (
-            ValueError,
-            pint.errors.DimensionalityError,
-            pint.errors.UndefinedUnitError,
-        ) as e:
+        except Exception:
```

Comment:
I don't like this blanket catching of all exceptions here, as it seems a little bug prone. If someone introduces an bug into `extract_model_time_details` then the error message will be a bit confusing.

I'd just use a custom exception type, as described above.